packages/fuels-core/src/lib.rs

```diff
@@ -427,6 +498,29 @@ impl Parameterize for u64 {
     }
 }
 
+impl<T> Parameterize for Option<T>
+where
+    T: Parameterize + Tokenizable,
+{
+    fn param_type() -> ParamType {
+        let types = vec![ParamType::Unit, T::param_type()];
+        let variants = EnumVariants::new(types).expect("should never happen");
```

Comment:
I'd explain in the expect message that since `!types.is_empty()` holds and `EnumVariants::new` fails only if it had no variants, this should never happen.